src/Internal.AspNetCore.Sdk/build/Git.targets

```diff
@@ -110,6 +110,10 @@ Generates an assembly attribute with commit hash
         <_Parameter1>CommitHash</_Parameter1>
         <_Parameter2>$(RepositoryCommit)</_Parameter2>
       </_CustomAttributes>
+      <_CustomAttributes Include="System.Reflection.AssemblyMetadataAttribute">
+        <_Parameter1>RepositoryCommitUrl</_Parameter1>
```

Comment:
If we include the RepositoryCommitUrl in the assembly metadata, is there a point in including the repository commit still or can we remove it?

Reply:
Hmm, keep both